mkenv/common.py

```diff
@@ -154,6 +154,10 @@ def convert(self, value, param, context):
     help="The directory to link scripts into.",
 )
 
+EX_OK = getattr(os, 'EX_OK', 0)
+EX_USAGE = getattr(os, 'EX_USAGE', 1)
+EX_NOINPUT = getattr(os, 'EX_NOINPUT', 1)
```

Comment:
Maybe you would like to pick arbitrary unique values for when the exit codes are not system provided?

Reply:
Yeah probably just using the numeric values on whatever system is a decent idea, I think Windows doesn't have any convention here (not that the *nix one is ever followed, so really in theory I shouldn't be so pedantic, but hey someone's gotta be).

E.g., EX_USAGE on this darwin box is 64, so maybe using that is reasonable.

Would like to keep these private though (so _EX_OK, etc.) so that they aren't misconstrued to be public API.